### PR TITLE
Dasd 7963 add pipeline

### PR DESCRIPTION
**Pull request by NickGraham101**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  value: 'SFA.DAS.EmployerIncentives'
+- name: BuildPlatform
+  value : 'any cpu'
+- name: BuildConfiguration
+  value: 'release'
+- group: 'Prod Management Resources'
+
+resources:
+  repositories:
+  - repository: self
```
Comment (jack-education) [inline azure-pipelines.yml:21]: Is self needed?

File: azure/template.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        "version": {
+            "type": "string",
+            "defaultValue": "1.0"
+        }
+    },
+    "variables": {
+        "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
+        "resourceNamePrefix": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', parameters('serviceName')))]",
+        "appServicePlanName": "[concat(variables('resourceNamePrefix'), '-asp')]",
+        "funcAppName": "[concat(variables('resourceNamePrefix'), '-as')]"
```
Comment (jack-education) [inline azure/template.json:62]:
```suggestion
        "funcAppName": "[concat(variables('resourceNamePrefix'), '-fa')]"
```

Comment (jack-education) [reply azure/template.json:62]: Can funcAppName be changed to functionAppName everywhere? I think it's more commonly used

File: azure/template.json [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+                    }
+                }
+            },
+            "dependsOn": [
+                "function-app"
+            ]
+        }
+    ],
+    "outputs": {
+        "funcAppName": {
```
Comment (jack-education) [inline azure/template.json:241]:
```suggestion
        "FunctionAppName": {
```
to match PascalCase for outputs https://github.com/SkillsFundingAgency/das-platform-building-blocks#property-name-formatting

Comment (NickGraham101) [reply azure/template.json]: changed

File: azure/template.json (hunk 2 above)
Comment (NickGraham101) [reply azure/template.json]: changed

File: azure-pipelines.yml (hunk 1 above)
Comment (NickGraham101) [reply azure-pipelines.yml]: removed

File: azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
```
Comment (jack-education) [inline azure-pipelines.yml:6]:
```suggestion
  pr: none
```
Pull request builds not needed as triggered on all branch commits
https://github.com/SkillsFundingAgency/das-reservations-api/blob/master/azure-pipelines.yml#L7

File: azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  pr: none
```
Comment (jack-education) [inline azure-pipelines.yml:6]:
```suggestion

pr: none

```

**Pull request merged**